### ISSUE TITLE
APPX -  HP REVERB G2 - WMR immersive - Crash at opening

### ISSUE THREAD

**Issue by sebstax7** [microsoft/MixedRealityToolkit-Unity#10030]
I built my application for WMR, using hp reverb G2, no error in my Unity log.

The application does not starts( I have a certificate issue at launch),  please my screen shot, I'm stuck at launch, the screen freeze and that's it.

## Expected behavior

the application should run

## Screenshots

![crash](https://user-images.githubusercontent.com/83447881/123431398-7f8caa80-d5c9-11eb-840a-91b19120ef38.PNG)

## Your setup (please complete the following information)

- Unity Version [2020.3.11]
- MRTK Version [2.7]

- WMR immersive
- OpenXR

**Comment by davidkline-ms** [comment]
The dialog appears to be asking for permission to use an application capability. Unfortunately, the dialog cuts off the specific capability being requested.

@keveleigh, please follow up to ensure that the proper versions of OpenXR, etc. are being used.

**Comment by keveleigh** [comment]
Hey @sebstax7! Can you please clarify which version of OpenXR you're using, as well as if you have the Mixed Reality OpenXR plugin installed (and which version)?

If you have the Mixed Reality OpenXR Plugin installed, can you confirm that you have the Mixed Reality Features feature enabled for UWP?

![image](https://user-images.githubusercontent.com/3580640/124009448-7cf7cf80-d992-11eb-9c58-97f16ad2d913.png)

**Comment by sebstax** [comment]
Hello @keveleigh @davidkline-ms, thanks for your support.

I finally got an APPX built by renaming in the Unity hierarchy "MRTK playspace" by "MAIN CAMERA".

Indeed, I added in "MRTKplayspace" my XRRIG and main camera otherwise, when teleporting (to move around in my application), controllers were stuck at their start coordinates and only my camera was moving.

Since this modification, my APPX build is working.

(My Unity Play log mentionned issue finding Main camera path)

Otherwise here are my answers to your questions : 

OpenXR is  version 1.1.1 with Unity 2020.3.7f1LTS
MR OpenXR is installed, version 0.9.5
MR OpenXR are well checked.

(It is a bit confusing to check "Microsoft Hololens" if working with mixed reality headset).
